lib/collection/src/operations/conversions.rs

```diff
@@ -777,6 +777,7 @@ impl TryFrom<api::grpc::qdrant::CollectionConfig> for CollectionConfig {
                 }
             },
             strict_mode_config: config.strict_mode_config.map(StrictModeConfig::from),
+            uuid: None,
```

Comment:
This is conversion from gRPC `CollectionConfig` into Qdrant `CollectionConfig`.

As far as I can tell, it's only used as part of `CollectionInfo` (which is transferred over gRPC), but never for anything important (like consensus related stuff), so it's fine not to include it into gRPC.

The only concern is if we **do** start using it for something important in the future, it might be easy to overlook. But I don't think it's very likely.

Reply:
What is the reason _here_ for not including it in gRPC. Just because that would expose it to end users?

Reply:
Yeah, pretty much. Seems easier to simply omit it here.